DD-784: load-from-fedora/vault: add File rights

jo-pol opened the pull request:
Fixes DD-784: load-from-fedora/vault: add File rights

# Description of changes
* read file rights from files.xml, 
* when not found, apply defaults from dataset.xml
  * [x] for load-from-vault
  * [x] for load-from-fedora, the columns `accessible_to` and  `visible_to` of `easy_files` are defined not-null. So it seems we don't need to read `dataset.xml`.
* new formatting for some classes lead to a lot of white space changes


# How to test

build, deploy and run on deasy: copy `src/test/resources/deasy.csv` to `easy-dtap/shared` and:

    dd-verify-file-migration load-from-fedora /vagrant/shared/deasy.csv



# Related PRs
(Add links)
*

# Notify
@DANS-KNAW/dataversedans

jo-pol committed:
read files.xml from vault

jo-pol committed:
refactoring

jo-pol committed:
simplified

jo-pol committed:
rewritten with SaxParser

jo-pol committed:
obsolete throws

jo-pol committed:
add rights columns to expected table

jo-pol committed:
license headers

jo-pol committed:
default file rights from dataset.xml

jo-pol committed:
formatting as far as more than leading white space

jo-pol committed:
configuration for deploy on datastation

janvanmansum reviewed (commented):
I put the comment in the wrong project, see https://github.com/DANS-KNAW/dd-dtap/pull/172#pullrequestreview-852113730

janvanmansum commented on src/main/java/nl/knaw/dans/filemigration/DdVerifyFileMigrationConfiguration.java:
```diff
@@ -51,6 +51,10 @@ public DataSourceFactory getEasyDb() {
+    this.easyDb = easyDb;
+  }
+
   @JsonProperty("verificationDatabase")
```
setVerificationDatabase below needs a capital V.

jo-pol committed:
no echo

asked by reviewer of https://github.com/DANS-KNAW/dd-dtap/pull/172

jo-pol committed:
camelcase

review https://github.com/DANS-KNAW/dd-verify-file-migration/pull/6#pullrequestreview-852120570
